SurveyApp/Web.config

```diff
@@ -0,0 +1,104 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+  For more information on how to configure your ASP.NET application, please visit
+  https://go.microsoft.com/fwlink/?LinkId=301880
+  -->
+<configuration>
+  <configSections>
+    <section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
+  <!-- For more information on Entity Framework configuration, visit http://go.microsoft.com/fwlink/?LinkID=237468 --></configSections>
+  <connectionStrings>
+    <add name="SurveyContext" connectionString="Data Source=WSC-138;Initial Catalog=SurveyContext;Integrated Security=True" providerName="System.Data.SqlClient" />
```

Comment:
Ideally, connection string should not be duplicated. Not it is in both Web and WebApi projects. It seems the connection string should be in a single reusable file.